solidity/contracts/converter/BancorConverterUpgrader.sol

```diff
@@ -69,17 +65,23 @@ contract BancorConverterUpgrader is IBancorConverterUpgrader, Owned, ContractIds
 
     /**
       * @dev initializes a new BancorConverterUpgrader instance
+      * 
+      * @param _registry    address of a contract registry contract
     */
-    constructor(IContractRegistry _registry) public {
-        registry = _registry;
+    constructor(IContractRegistry _registry) ContractRegistryClient(_registry) public {
     }
 
     /**
       * @dev allows the owner to update the contract registry contract address
       * 
       * @param _registry   address of a contract registry contract
     */
-    function setRegistry(IContractRegistry _registry) public ownerOnly {
+    function setRegistry(IContractRegistry _registry)

```

Comment:
I think we can probably live without this function and rely on updateRegistry if we need to.
We can also always just deploy a new upgrader and update the registry if we really have to

Reply:
OK, so remove it from both `BancorNetwork` and `BancorConverterUpgrader`?

Reply:
yup

Reply:
Done